src/auditlog/backends/base.py

```diff
@@ -0,0 +1,51 @@
+from auditlog.diff import model_instance_diff
+from auditlog.models import LogEntry
+
+
+class AuditBackend(object):
+    """Base backend for creation of audit log entries."""
+
+    def log_create(self, instance, created, **kwargs):
+        """Handle model creation logging"""
+        if created:
+            changes = model_instance_diff(None, instance)
+
+            self.create_log(
+                instance=instance,
+                action=LogEntry.Action.CREATE,
+                changes=changes,
+                **kwargs)
+
+    def log_update(self, instance, **kwargs):
+        """Handle model update logging"""
+        if instance.pk is not None:
+            try:
+                old = instance.__class__.objects.get(pk=instance.pk)
+            except instance.__class__.DoesNotExist:
+                pass
```

Comment:
Silent failure here.

Reply:
@bujniewicz have to leave this as its from their codebase and i don't want to break backwards compatibility.

Reply:
ugh, this sucks :P